CNV/cnv-2.1.0.sh

```diff
@@ -154,3 +154,141 @@ metadata:
   namespace: "${TARGET_NAMESPACE}"
 spec: {}
 EOF
+
+echo "Waiting for HCO to get fully deployed"
+oc wait -n ${TARGET_NAMESPACE} hyperconverged hyperconverged-cluster --for condition=Available --timeout=10m
+
+echo "Applying new network configuration on all nodes, one by one, in order to sustain quorum"
+machineCIDR=$(grep 'machineCIDR' ../OpenShift/install-config.yaml | sed 's/\(.*\): *\(.*\)/\2/')
+storageCIDR="172.22.0.0/24"  # TODO: get from a config file
```

Comment:
Note this subnet is reserved for provisioning, and the only nic in the deployment which is up on this network is the node where the baremetal-operator is running, so you probably need to choose a different range and configure storage traffic to go via a vlan - I think @sreichar has a spreadsheet with vlan IDs and subnet ranges to refer to?

Reply:
On my libvirt setup, this subnet is available on all nodes on the secondary interface. storageCIDR is indeed a bad name for this. I will drop VLAN configuration from this PR and limit it to bridge, so we get something working in first. I'd like to sync with you folks to discuss what should be done with the VLAN part.